packages/certs-v5/commands/certs/add.js

```diff
@@ -199,10 +200,41 @@ function * addDomains (context, heroku, meta, cert) {
   }
 }
 
+function * configureDomains (context, heroku, meta, cert) {
+  let certDomains = cert.ssl_cert.cert_domains
+  let apiDomains = yield waitForDomains(context, heroku)
+  let appDomains = apiDomains.map(domain => domain.hostname)
+  let matchedDomains = matchDomains(certDomains, appDomains)
+
+  if (matchedDomains.length > 0) {
+    cli.styledHeader('Almost done! Which of these domains on this application would you like this certificate associated with?')
+
+    let selectedDomains = (yield inquirer.prompt([{
+      type: 'checkbox',
+      name: 'domains',
+      message: 'Select domains',
+      choices: matchedDomains
+    }])).domains
+
+    if (selectedDomains.length > 0) {
+      yield Promise.all(selectedDomains.map(domain => {
+        heroku.request({
```

Comment:
You need to `return` here in order for `Promise.all` to work since it needs an array of actual Promises